server.py

```diff
@@ -1184,6 +1192,13 @@ async def losses_cmd(ctx: ContextType):
         losses = [int(x) for x in f.read().split()]
     await ctx.reply(msg.format(losses, sum(losses)))
 
+@command("lossestest")
+async def lossestest_cmd(ctx: ContextType):
```

Comment:
Also need renaming here!

Reply:
oh I wanted to keep this... I don't test the disc/twitch commands locally, or at least don't really know how to. So I'm concerned about overwriting Baalor's commands without them being tested. Unless you'll test them before deploying the changes?

Reply:
You can keep the code but just comment out the `@command` before the old ones, and name them the same. That way if there's an issue it can easily be reverted.